duckduckgo/iOS
# Update Autofill Logins copy

Task/Issue URL: https://app.asana.com/0/0/1205931258794163/f
Tech Design URL:
CC:

**Description**:
Updates all uses of the word 'Logins' with the word 'Password'



**Steps to test this PR**:

**Note:** Translations will be tackled in a separate PR
1. Confirm autofill strings have been updated as per this [sheet](https://duckduckgo.sharepoint.com/:x:/r/sites/O-I/Shared%20Documents/Change%20logins%20to%20passwords.xlsx?d=w0326563c3d844109b1a928d278b44576&csf=1&web=1&e=ShsMTM)




**Copy Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
